x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/connector_step/connector_step.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { useLoadConnectors } from '@kbn/elastic-assistant';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiPopover, EuiLink } from '@elastic/eui';
+import { useKibana } from '../../../../../common/hooks/use_kibana';
+import { StepContentWrapper } from '../step_content_wrapper';
+import { ConnectorSelector } from './connector_selector';
+import { ConnectorSetup } from './connector_setup';
+import type { AIConnector } from '../../types';
+import { useActions } from '../../state';
+import * as i18n from './translations';
+
+/**
+ * List of allowed action type IDs for the integrations assistant.
+ * Replace by ['.bedrock', '.gen-ai'] to allow OpenAI connectors.
```

Comment:
Plans to support `.gemini`?

Reply:
Yes, but not for 8.15. 
Right now we only have Bedrock available since the workflow and prompts have been fine-tuned for Opus, but the Integrations team is working on adding support for other models, OpenAI will be first. Gemini will come right after, I assume.